common/src/main/java/fi/hel/verkkokauppa/common/rest/CommonServiceConfigurationClient.java

```diff
@@ -0,0 +1,34 @@
+package fi.hel.verkkokauppa.common.rest;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+
+
+@Component
+public class CommonServiceConfigurationClient {
+    
+    private Logger log = LoggerFactory.getLogger(CommonServiceConfigurationClient.class);
+
+    @Autowired
+    private Environment env;
+
+    @Autowired
+    private RestServiceClient restServiceClient;
+
+    public String getRestrictedServiceConfigurationValue(String namespace, String key) {
+        String serviceMappingUrl = env.getRequiredProperty("serviceconfiguration.url") + "restricted/get?namespace=" + namespace + "&key=" + key;
```

Comment:
should inject serviceconfiguration.url as class variable (@Value) so it gets read at component startup

Reply:
- [ ] good point, I will add that as @value.